cnf-certification-test/certification/suite.go

```diff
@@ -150,3 +151,28 @@ func testHelmCertified(env *provider.TestEnvironment) {
 		ginkgo.Fail(fmt.Sprintf("%d helms chart are not certified.", len(failedHelmCharts)))
 	}
 }
+func testContainerCertificationStatusByDigest(env *provider.TestEnvironment) {
+	containersToQuery := certtool.GetContainersToQuery(env)
```

Comment:
we need a need GetContainersToQuery to return only containers that are running on the cluster. Maybe pass a parameter "skipConfigFile"?